docs/reference/deprecated.md

```diff
@@ -14,6 +14,7 @@ Gutenberg's deprecation policy is intended to support backwards-compatibility fo
 
  - `wp.components.withContext` has been removed. Please use `wp.element.createContext` instead. See: https://reactjs.org/docs/context.html.
  - `wp.coreBlocks.registerCoreBlocks` has been removed. Please use `wp.blockLibrary.registerCoreBlocks` instead.
+- `wp.components.Draggable` as a DOM node drag handler has been deprecated. Please, use `wp.components.Draggable` as a wrap component for your DOM node drag handler.
```

Comment:
Is there a better way to explain this change?

Reply:
This deprecation will be removed in 4.0 release if it is planned for 3.8 release.

Reply:
Do we need to include the version that will remove the behavior here? I haven't seen it anywhere else.

Reply:
Just to make sure: the deprecation is added to the version it was deprecated, right? Or should we move this to the `4.0` section?

Reply:
> The current deprecations are listed below and are grouped by _the version at which they will be removed completely_.

It should be under 4.0 😄

Reply:
:writing_hand:  32006d3